components/cdc/src/delegate.rs

```diff
@@ -394,11 +389,7 @@ impl Delegate {
         for lock in pending.take_locks() {
             match lock {
                 PendingLock::Track { key, start_ts } => resolver.track_lock(start_ts, key),
-                PendingLock::Untrack {
-                    key,
-                    start_ts,
-                    commit_ts,
-                } => resolver.untrack_lock(start_ts, commit_ts, key),
+                PendingLock::Untrack { key } => resolver.untrack_lock(&key),
```

Comment:
Better to keep `start_ts` and `commit_ts`, it could help debugging.

Reply:
But actually we don't have ways to know them.

Reply:
Just in the case of deleting lock?

Reply:
Yes. Locks are all untracked by deleting lock events.

Reply:
hmmm, got.